src/typings.ts

```diff
@@ -374,14 +375,14 @@ export interface Signature {
   concatSig: string
 }
 
-export interface IdentityBackup {
+export interface DeserialisedIdentityWallet {
```

Comment:
That does not make sense. I dont know what a deserializedWallet should be.

Reply:
Maybe `IdentityWalletSerializationFormat` ?

Reply:
I don't know what this means.